app/views/admin/users/edit.html.erb

```diff
@@ -55,13 +55,13 @@
     <div class="label-input">
       <%= f.label :password %>
       <br>
-      <%= f.password_field :password, required: true %>
+      <%= f.password_field :password, value: @user.password_digest %>
```

Comment:
you generally don't want to put their digest here ... if a user has a short password and then they see it filled in with something that's 40-something bytes long they'll get confused... plus the password digest would then be stored as their actual password.